public/docs/dart/latest/guide/displaying-data.jade

```diff
@@ -1,472 +1,265 @@
-   insert it into the browser,
-   automatically updating it whenever it changes.
-
-  .callout.is-helpful
-    header Note
-    p.
-      While you've used <code>template:</code> to specify an inline view, for larger templates you'd
-      want to move them to a separate file and load them with <code>templateUrl:</code> instead.
-
-  p.
-   One thing to notice is that although you've written
-   your <code>DisplayComponent</code> class, you haven't
-   used <code>new</code> to instantiate it.
-   Because your class is associated with <code>&lt;display&gt;</code> elements in
-   the DOM, Angular automatically calls <code>new</code> on
-   <code>DisplayComponent</code> and bind its properties to
-   that part of the template.
-
-  p.
-   When you're building templates, data bindings like these have access to
-   the same scope of
-   properties as your controller class does.
-   Here your class is <code>DisplayComponent</code>, which has
-   just one property, <code>myName</code>.
-
-  p.
-    Add a second line to the template,
-    so you can see Angular dynamically update content:
-
-  code-example(language="html").
-    &lt;p&gt;Current time: {{ time }}&lt;/p&gt;
-
-  p.
-    Then import <code>dart:async</code> so you can use a Timer,
-    and give the <code>DisplayComponent</code> a starting value for time and
-    create a periodic Timer call to update the time:
-
-  code-example(language="dart").
-    import 'dart:async';
-    ...
-    class DisplayComponent {
-      String myName = 'Alice';
-      String time;
-      Timer _timer;
-
-      DisplayComponent() {
-        _updateTime(null);
-        _timer = new Timer.periodic(new Duration(seconds: 1), _updateTime);
-      }
-
-      _updateTime(Timer _) {
-        time = new DateTime.now().toString();
-      }
-    }
-
-  p Reload the app, and you'll now see the seconds updating automatically.
-
-<a id="ng-for"></a>
+:marked
+  ## Showing component properties with interpolation
+  The easiest way to display a component property
+  is to bind the property name through interpolation.
+  With interpolation, we put the property name in the view template, enclosed in double curly braces: `{{myHero}}`.
+
+  Let's build a small illustrative example together.
+
+  <a id="platform_directives"></a>
+  Create a new project folder (`displaying`) and create 3 files:
+  `pubspec.yaml`, `web/index.html`, and `web/main.dart`.
+  Put these contents in the files:
+
+- var stylePattern = [{ otl: /(platform_directives.*$)/gm }, null, null];
++makeTabs('displaying-data/dart/pubspec.yaml, displaying-data/dart/web/index.html, displaying-data/dart/web/main.dart', ',,final', 'pubspec.yaml, web/index.html, web/main.dart', stylePattern)
+
+:marked
+  All of this code should look familiar from the
+  [QuickStart](../quickstart.html),
+  except for the `platform_directives` entry in `pubspec.yaml`
+  and the imports in `main.dart`.
+
+  In `pubspec.yaml`, the `platform_directives` entry lets us use
```

Comment:
Is it possible to link to documentation on the transformer here for people who are interested further? This is definitely new to me :)